plugins/nexus-repository-manager/app-config.janus-idp.yaml

```diff
@@ -0,0 +1,27 @@
+proxy:
+  endpoints:
+    '/nexus-repository-manager':
+      target: 'https://<NEXUS_REPOSITORY_MANAGER_URL>'
```

Comment:
As in previous comments, this will override the [showcase `app-config.yaml` settings](https://github.com/janus-idp/backstage-showcase/blob/main/app-config.yaml#L102-L110) with a default URL that doesn't work.